<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_CardLbl</class>
 <widget class="QWidget" name="Widget_CardLbl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>68</width>
    <height>108</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="Widget_Main" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#Widget_Main{
	border: 1px solid balck;
	border-radius: 3px;
	background-color: rgb(255, 255, 250);
}

QLabel{	
	font: 18pt;
}</string>
     </property>
     <widget class="QLabel" name="lbl_rank">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>10</y>
        <width>24</width>
        <height>91</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_suit">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>40</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_suit_big">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
